.github/workflows/ci.yml

```diff
@@ -542,6 +542,65 @@ jobs:
             build/*.cmake
             build/testsuite/*/*.*
 
+  icx-vp2022:
+    name: "Linux VFXP-2022 icx/C++17 llvm13 py3.9 boost1.76 exr3.1 oiio2.3 avx2"
+    runs-on: ubuntu-latest
+    container:
+      image: aswftesting/ci-osl:2022-clang13
+    env:
+      CXX: icpx
+      CC: icx
+      CMAKE_CXX_STANDARD: 17
+      FMT_VERSION: 7.1.3
+      OPENIMAGEIO_VERSION: master
+      OPENIMAGEIO_CMAKE_FLAGS: -DBUILD_FMT_VERSION=7.1.3
+      PYBIND11_VERSION: v2.9.0
+      PYTHON_VERSION: 3.9
+      USE_BATCHED: b8_AVX2_noFMA
+      # USE_BATCHED: b16_AVX512
+      # USE_SIMD: avx2,f16c
```

Comment:
I had commented out the `USE_SIMD` for icc because it uses a different syntax than the `-m` arguments that this gets turned into by compiler.cmake. But icx accepts the same `-m` arguments as clang. So you can re-enable that line for icx.